modules/pel/peltool/config.py

```diff
@@ -16,3 +16,5 @@ def __init__(self) -> None:
         self.only = False
         self.plid = None
         self.src = None
+        self.bmcID = None
```

Comment:
Commit message:

Header is not articulating the problem.

`peltool: Fix, Get NonServiceable PEL by --id/--bmc-id `